benches/src/lib.rs

```diff
@@ -226,11 +235,20 @@ impl VmBench {
         self
     }
 
+    /// Replaces the current prepare script with the given one.
+    /// Not that if you've constructed this instance with `contract` or `using_contract_db`,
+    /// then this will remove the script added by it. Use `extend_prepare_script` instead.
     pub fn with_prepare_script(mut self, prepare_script: Vec<Instruction>) -> Self {
         self.prepare_script = prepare_script;
         self
     }
 
+    /// Adds more instructions before the current prepare script.
+    pub fn prepend_prepare_script(mut self, prepare_script: Vec<Instruction>) -> Self {
+        self.prepare_script.extend(prepare_script);
+        self
+    }
+
```

Comment:
It seems you can use `with_prepare_script` here because `prepare_script` is empty.
<img width="801" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/6fd4c57b-4fbe-421b-af4a-c7c138fa3037">

Reply:
How is it empty? `contract_using_db` sets `prepare_script` to

https://github.com/FuelLabs/fuel-core/blob/5c45051eee3d8df9719ffd5d42c3aafe6223e634/benches/src/lib.rs#L184-L190